docs/code-style.md

```diff
@@ -13,22 +13,39 @@ guide](https://github.com/excelmicro/typescript) is a good place to look.
 > Why? For components, a default export will show as `<Unknown>` in the React dev tools unless explicity declared to a constant first. Also, default exports can lead to confusing situations. (Using the name of the export as a local variable in the file, exporting named and a default)
 
 ```js
-//bad
-//MyComponent.tsx
-export default () => <h1>My component</h1>
+// bad
+// MyComponent.tsx
+export default () => <h1>My component</h1>;
 
 //index.tsx
-import MyComponent from './MyComponent'
+import MyComponent from './MyComponent';
 
 ```
 
 ```js
-//good
-//MyComponent.tsx
-export const MyComponent = () => <h1>MyComponent</h1>
+// good
+// MyComponent.tsx
+export const MyComponent = () => <h1>MyComponent</h1>;
 
-//index.tsx
-import {MyComponent} from './MyComponent'
+// index.tsx
```

Comment:
Taking into account the next section of the docs maybe we should rename this example module to something other than `index.tsx`?